src/pages/dar_application/DucAddendum.js

```diff
@@ -133,7 +133,7 @@ export default function DucAddendum(props) {
             style: columnStyles
           },
           {
-            data: dataset.datasetName.replaceAll("_", "_\u200b"),
+            data: dataset.datasetName.replaceAll('_', '_\u200b'),
```

Comment:
Technically it is still possible to have a really really long alphanumeric name and this wouldn't cause a line break, but this seems like a simple way to fix the vast majority of cases.

Reply:
Makes sense, i'm happy with this solution.